src/Illuminate/Database/Eloquent/Builder.php

```diff
@@ -1053,11 +1055,23 @@ public function increment($column, $amount = 1, array $extra = [])
      */
     public function decrement($column, $amount = 1, array $extra = [])
     {
-        return $this->toBase()->decrement(
-            $column, $amount, $this->addUpdatedAtColumn($extra)
+        return $this->update(
+            $this->toBase()->createDecrementStatement(
```

Comment:
```suggestion
            $this->createDecrementStatement(
```
You already have the `toBase()` call inside of `createDecrementStatement()`